cooltools/cli/compute_expected.py

```diff
@@ -0,0 +1,121 @@
+import click
+
+import cooler
+
+from . import cli
+from .. import expected
+
+
+
+# might be relevant to us ...
+# https://stackoverflow.com/questions/46577535/how-can-i-run-a-dask-distributed-local-cluster-from-the-command-line
```

Comment:
hmm...this suggests that if someone has launched their own dask scheduler as a daemon process, we could provide an option to pass in the `<host>:<port>` to the scheduler instead of starting a new cluster ourselves.